File: front/src/Interfaces/CartItemModel.ts
// export interface CartItemModel {
//   bookId: string;
//   quantity: number;
//   title: string;
//   category: string;
//   price: number;
// }
import { BookModel } from "./BookModel";

export interface CartItemModel {
  photoUrl:string;
  bookId: string;
  quantity: number;
  title: string;
  category: string;
  price: number;
}

File: Domain/Model/CartItemModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Model
{
    public class CartItemModel
    {
        public Guid BookId { get; set; }
        public int Quantity { get; set; }
        public string Title { get; set; }
        public string Category { get; set; }
        public decimal Price { get; set; }
        public string PhotoUrl { get; set; }
    }
}
